crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -117,6 +118,8 @@ pub struct BaseRoomInfo {
     /// others, and this field collects them.
     #[serde(skip_serializing_if = "RoomNotableTags::is_empty", default)]
     pub(crate) notable_tags: RoomNotableTags,
+    /// The `m.room.pinned_events` of this room.
+    pub(crate) pinned_events: Option<RoomPinnedEventsEventContent>,
```

Comment:
Note to self: i think we could have used `MinimalStateEvent` here, but it's unclear what's the value of it, so no need to change anything here.

Reply:
I can try changing it so it matches the other fields if it's easy to do (I'm not sure how I missed it 🫤 ).

Reply:
Actually, it does make everything way more complex just to store a possible event id which isn't used anywhere, so it's probably not worth to add it.